lib/abstract_format.ml

```diff
@@ -941,6 +984,15 @@ and guard_test_assoc_of_sf sf : (guard_test_assoc_t, err_t) Result.t =
   | _ ->
      Err.create ~loc:[%here] (Err.Not_supported_absform ("guard_test_assoc", sf)) |> Result.fail
 
+and atom_or_wildcard_of_sf sf =
+  match sf with
+  | Sf.Tuple (3, [Sf.Atom "atom"; _; Sf.Atom field_name]) ->
+     AtomWildcardAtom field_name |> Result.return
+  | Sf.Tuple (3, [Sf.Atom "var"; _; Sf.Atom "_"]) ->
+     AtomWildcardWildcard |> Result.return
+  | _ ->
+     Err.create ~loc:[%here] (Err.Not_supported_absform ("atom_or_wildcard", sf)) |> Result.fail
```

Comment:
Could you use `Err.Invalid_input` because there are no patterns except for the above 2 patterns?

```suggestion
     Err.create ~loc:[%here] (Err.Invalid_input ("invalid form of atom_or_wildcard", sf)) |> Result.fail
```

Reply:
yes